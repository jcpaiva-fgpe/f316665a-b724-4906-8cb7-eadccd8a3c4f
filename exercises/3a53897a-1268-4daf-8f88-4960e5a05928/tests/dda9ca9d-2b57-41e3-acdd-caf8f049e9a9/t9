-40000 -25000 4000 35000
1
1000 1000
