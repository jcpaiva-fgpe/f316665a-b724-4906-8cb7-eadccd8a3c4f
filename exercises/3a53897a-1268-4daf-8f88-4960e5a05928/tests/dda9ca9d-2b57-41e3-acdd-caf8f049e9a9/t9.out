8
Barcelona 1
Frankfurt 1
Funchal 1
Lisboa 1
Londres 1
Madrid 1
Paris 1
Porto 1
